### PR TITLE
Enabled schema registry and ksql server.

### PR DESCRIPTION
**Pull request by jacobcrawford**
The kafkadb-server and schema registry has been enabled from the Confluent helm repository.

The enabelling of the kafkadb-server makes it possible to create Kafka Streams and Tables based on topics that are reflected in the tests. The tests use a Python wrapper around the kafkasql API of the server.

### CODE REVIEW

File: .github/workflows/test_create.yaml [hunk 1]
```diff
@@ -47,3 +44,11 @@ jobs:
...
         run: helm test test-kafka --filter name=test-kafka-example-service
 
+      - name: Wait for ksql-server
+        run: |
+         ./chart/scripts/wait_for_pod.sh kafka-cp-ksql-server
+
+      - name: Helm test Kafka ksql
+        run: helm test test-kafka --filter name=test-kafka-ksql
+
+
```
Comment (klausenbusk) [inline .github/workflows/test_create.yaml:54]: Let's remove these two blank lines.

File: chart/scripts/python_ksql.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
+
+from ksql import KSQLAPI
```
Comment (klausenbusk) [inline chart/scripts/python_ksql.py:2]:
```suggestion
from ksql import KSQLAPI
```

File: docs/ksql-examples.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Create stream from a kafka topic
+```
```
Comment (klausenbusk) [inline docs/ksql-examples.md:2]:
```suggestion
# Create stream from a kafka topic
```sql
```
Add sql here and for the other examples for syntax highlighting.

Ex:
```sql
CREATE STREAM test_stream_1 (number INT) 
    WITH (KAFKA_TOPIC='topic-test-1', VALUE_FORMAT='JSON'); 
```

**Review by skovbakke — COMMENTED** [review]
Just a few comments for now

File: chart/templates/tests/test-kafka-ksql-pod.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
+{{ if .Values.env }}
+{{ if eq .Values.env "test" }}
```
[the remark this answers is not in the document]
Comment (jacobcrawford) [reply chart/templates/tests/test-kafka-ksql-pod.yaml]: I guess you are right :)

**Review by klausenbusk — CHANGES_REQUESTED** [review]
Minor comment, but LGTM 🎉

**Review by klausenbusk — APPROVED** [review]
LGTM 🎉

**Pull request merged**